conf/config.yaml.example

```diff
@@ -674,6 +674,7 @@ deployment:                    # Deployment configurations
       #   role: viewer
 
     enable_admin_cors: true       # Enable Admin API CORS response header `Access-Control-Allow-Origin`.
+    enable_admin_ui: true         # Enable embedded APISIX Dashboard UI.
```

Comment:
if the role of APISIX is data plane, will `enable_admin_ui` be false automatically? Or we can set `enable_admin_ui` to false by default.

Reply:
So I don't think anything needs to be done about it.